cardano-node/src/Cardano/Node/Run.hs

```diff
@@ -379,7 +381,7 @@ handleSimpleNode p trace nodeTracers npm onKernel = do
                         -> ChainDB.ChainDbArgs IO blk
                         -> ChainDB.ChainDbArgs IO blk
    customiseChainDbArgs runValid args = args
-     { ChainDB.cdbValidation = if runValid
+     { ChainDB.cdbImmValidation = if runValid
```

Comment:
Can you get add `ChainDB.cdbVolValidation` too? See https://github.com/input-output-hk/ouroboros-network/blob/f5741efd37c5558d8a29d0d2b142e01f650a3d45/ouroboros-consensus/src/Ouroboros/Consensus/Node.hs#L150.

For example:
```haskell
customiseChainDbArgs runValid args
  | runValid
  = args
    { ChainDB.cdbImmValidation = ValidateAllEpochs
    , ChainDB.cdbVolValidation = ValidateAll
    }
  | otherwise
  = args
```

Reply:
Please don't forget to add this.